src/main/java/org/jenkinsci/plugins/ewm/steps/ExwsAllocateExecution.java

```diff
@@ -62,15 +62,15 @@ protected ExternalWorkspace run() throws Exception {
             Disk disk = allocationStrategy.allocateDisk();
 
             String physicalPathOnDisk = disk.getPhysicalPathOnDisk();
-            if (physicalPathOnDisk == null) {
-                String message = format("Physical path on disk was not provided in the Jenkins global config for the Disk Pool ID '%s'", diskPoolId);
-                throw new AbortException(message);
-            }
             String diskId = disk.getDiskId();
             if (diskId == null) {
                 String message = format("Disk ID was not provided in the Jenkins global config for the Disk Pool ID '%s'", diskPoolId);
                 throw new AbortException(message);
             }
+            if (physicalPathOnDisk == null) {
```

Comment:
No, I have just changed the order of null checks.
When I was writing the tests, I though it was more appropriate to check firstly if the diskId is null, and the check if the physicalPathOnDisk is null.
Before, I was checking if the physicalPathOnDisk is null, and afterwards if the diskId is null.